docs/src/main/asciidoc/cdi-reference.adoc

```diff
@@ -200,43 +200,20 @@ public class CounterBean {
 ----
 <1> A package-private constructor injection. `@Inject` is optional in this particular case.
 
-[[supported_features]]
-== Supported Features
-
-* Programming model
-** Managed beans implemented by a Java class
-*** `@PostConstruct` and `@PreDestroy` lifecycle callbacks
-** Producer methods and fields, disposers
-** Qualifiers
-** Alternatives
-** Stereotypes
-* Dependency injection and lookup
-** Field, constructor and initializer/setter injection
-** Type-safe resolution
-** Programmatic lookup via `jakarta.enterprise.inject.Instance`
-** Client proxies
-** Injection point metadata
-* Scopes and contexts; see xref:cdi.adoc#bean-scope-available[here] for details
-* Interceptors
-** Business method interceptors: `@AroundInvoke`
-** Interceptors for lifecycle event callbacks: `@PostConstruct`, `@PreDestroy`, `@AroundConstruct`
+[[supported_features_and_limitations]]
```

Comment:
I wonder if we should also [keep the original section ids](https://docs.asciidoctor.org/asciidoc/latest/sections/custom-ids/#assign-auxiliary-ids) so that we don't break existing links...

Reply:
Interesting, I didn't know about that. I'll fix this.